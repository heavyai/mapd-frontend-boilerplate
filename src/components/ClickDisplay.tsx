import React from 'react'

import { ClicksState } from '~/reducers/clicks'


type Props = {
  clicks: ClicksState
}

const ClickDisplay: React.StatelessComponent<Props> = ( { clicks: { total } } ) =>
  <div>
    You have clicked the button { total } times
  </div>


export default ClickDisplay
