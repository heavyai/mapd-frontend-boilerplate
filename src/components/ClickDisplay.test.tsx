import React from 'react'
import * as renderer from 'react-test-renderer'

import ClickDisplay from './ClickDisplay'


test( 'Component renders', () => {
  const component = renderer.create(
    <ClickDisplay clicks={ { total: 1 } } />
  )

  expect( component.toJSON() ).toMatchSnapshot()
} )
