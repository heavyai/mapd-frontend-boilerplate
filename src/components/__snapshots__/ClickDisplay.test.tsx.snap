// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders 1`] = `
<div>
  You have clicked the button 
  1
   times
</div>
`;
